datajoint/expression.py

```diff
@@ -73,6 +75,17 @@ def heading(self):
         """a dj.Heading object, reflects the effects of the projection operator .proj"""
         return self._heading
 
+    @property
+    def top(self):
```

Comment:
I am not sure if parsing `self._top` is ncessary every time `self.top` is evoked. I would just the entire `dj.Top` object and parse it once at SQL generation time.